packages/core/src/components/forms/_controls.scss

```diff
@@ -197,7 +197,6 @@ $control-indicator-spacing: $pt-grid-size !default;
 
   input:focus ~ .#{$ns}-control-indicator {
     @include focus-outline();
-    outline: $blue3 solid 2px;
```

Comment:
from what I can tell this was just to provide increased contrast, no reason to not use the consistent `focus-outline` style now